pages/blog.html

```diff
@@ -3,23 +3,34 @@
 title: Blog
```

Comment:
>Blog [](http://example.com/codeflow?start=7&length=4)

The latest blog shown on the page is from 2018, which makes the technology look old and unused. As the comment says we have been adding blogs to another site.  There are a couple interesting blogs we can add (OASIS releases Data Aggregation extensions, use of OData by BurnRate, etc.). 

I wonder, though, if there's a way to bring the blogs from the other site inline here.